libraries/AP_GPS/AP_GPS_SBP.cpp

```diff
@@ -289,6 +289,7 @@ AP_GPS_SBP::_attempt_state_update()
 
         last_full_update_tow = last_vel_ned.tow;
         last_full_update_cpu_ms = now;
+        state.rtk_age_cs       = (last_full_update_cpu_ms - last_injected_data_ms) / 10;
```

Comment:
I don't think this is really a fair metric. It might be sorta correct, but provides no guarantees that the needed data was injected, or that the data you have scheduled for injection has been injected yet (or processed).